packages/terra-clinical-result/src/ClinicalResult.jsx

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl, intlShape } from 'react-intl';
+import classNames from 'classnames/bind';
+import ClinicalResultDisplay from './_ClinicalResultDisplay';
+import ResultError from './common/other/_ResultError';
+import NoData from './common/other/_KnownNoData';
+import statusPropType from './proptypes/statusPropTypes';
+import interpretationPropType from './proptypes/interpretationPropTypes';
+import valueQuantityPropType from './proptypes/valuePropTypes';
+import styles from './ClinicalResult.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * A unique event identifier attached to the result data
+   */
+  eventId: PropTypes.string,
```

Comment:
Should this identifier be required?

Reply:
For all normal results it would be expected to provide an id, but since the same component has built in displays for “error” and known “no data” scenarios, those are some scenarios for which it wouldn’t be needed, going against setting it as required. We had a discussion about it specifically and initial thoughts were to keep it non-required for the time being, monitor usage, and then major version bump later with any other adjustments if it is determined as needed.